src/main/java/org/ice4j/ice/harvest/SinglePortUdpHarvester.java

```diff
@@ -303,21 +302,19 @@ private void runInHarvesterThread()
                     catch (SocketException se)
                     {
                         logger.info("Could not create a socket: " + se);
-                        continue;
```

Comment:
This is not a critique, and I don't expect this addressed in any way. I'm just curious: Is there any specific reason these were removed? Syntactically, they don't do anything (or is there something subtle I'm missing?), but I think they give a hint to anyone reading the code about its intention.